default/config.yaml

```diff
@@ -4,15 +4,26 @@ dataRoot: ./data
 # -- SERVER CONFIGURATION --
 # Listen for incoming connections
 listen: false
+# Enables IPv6 and/or IPv4
+protocol:
+    ipv4: true
+    ipv6: false
+# Prefers IPv6 for dns, you should probably enable this on ISPs that don't have issues with IPv6
+dnsPreferIPv6: false
+# the hostname that autorun opens probably best left on auto. use options like 'localhost', 'st.example.com'
+autorunHostname: "auto"
 # Server port
 port: 8000
+# overrides the port for autorun with open your browser with this port and ignore what port the server is running on. -1 is use server port
+autorunPortOverride: -1
```

Comment:
since i added an option to change the hostname which could be used to point to any hostname or domain i added the port override if you want autorun to open a reverse proxy, e.g. you have a reverse proxy set up for st.example.com port 443 to go to st.internal.example.org port 8000